Formula/apache-flink.rb

```diff
@@ -1,25 +1,55 @@
 class ApacheFlink < Formula
   desc "Scalable batch and stream data processing"
   homepage "https://flink.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=flink/flink-1.3.2/flink-1.3.2-bin-hadoop27-scala_2.11.tgz"
-  version "1.3.2"
-  sha256 "378be2346990264eeff2ae012175523fc94ddbe59bead7e2a76e0d93c0c7c455"
+  url "https://www.apache.org/dyn/closer.lua?path=flink/flink-1.5.0/flink-1.5.0-bin-hadoop27-scala_2.11.tgz"
+  version "1.5.0"
+  sha256 "9a8232c124c80ea447cbc2c8edcaa0b4850aecaa9d506169c968777956ccd8ff"
   head "https://github.com/apache/flink.git"
 
   bottle :unneeded
 
-  depends_on :java => "1.7+"
+  depends_on :java => "1.8"
 
   def install
     rm_f Dir["bin/*.bat"]
     libexec.install Dir["*"]
-    bin.write_exec_script Dir["#{libexec}/bin/flink"]
+    (libexec/"bin").env_script_all_files(libexec/"libexec", Language::Java.java_home_env("1.8"))
+    chmod 0755, Dir["#{libexec}/bin/*"]
+    bin.write_exec_script "#{libexec}/bin/flink"
   end
 
   test do
+    input_file, output_file, expected_output = setup_test_environment
+    start_flink_cluster
+    execute_word_count(input_file, output_file)
+    stop_flink_cluster
+    output = File.open(output_file).read
+    assert_equal(expected_output, output)
+  end
+
+  def setup_test_environment
```

Comment:
let's avoid the separate functions. inlining everything within `test do` is preferable in terms of formula style

Reply:
Alright, I'll change it.